### ISSUE TITLE
Cursor position after wrapping

### ISSUE THREAD

**Issue by lukechampine** [stkb/vscode-rewrap#18]
IIRC the behavior of Alt-Q in SublimeText is to move the cursor along with the wrapped text. e.g.:

```
(before wrap)
foo bar baz |quux

(after wrap)
foo bar
baz |quux
```

I like this behavior because it allows me to continue typing where I left off. rewrap instead positions the cursor after `bar`, so I have to reposition.

It would be nice if the default behavior were changed, but I'd be perfectly happy with a config option as well.

**Comment by stkb** [comment]
Yeah I agree the cursor (and selection)  position should stay the same relative to the word it was next to.

While I can't reproduce your exact example, I know this is something that doesn't always work correctly. It's going to be tricky to fix but I will try.

**Comment by stkb** [comment]
Just published a new version (1.0.0); hopefully this should now be fixed. It seems to work correctly with everything I've thrown at it.

**Comment by lukechampine** [comment]
Confirmed working for me. Thanks stkb!
